### ISSUE TITLE
Envoy CVEs

### ISSUE THREAD

**Issue by stevesloka** [projectcontour/contour#2673]
Envoy has additional security CVEs included in v1.15.0 & v1.14.4. We should release a Contour v1.6.2 to include Envoy v1.14.4 and then release Contour v1.7 which will should upgrade to Envoy v1.15.0. 

Ref: https://groups.google.com/g/envoy-users/c/pvmKpPBHEcU/m/FFt7HBonAQAJ

**Comment by skriss** [comment]
Just curious, why do we need to do a Contour release in order to bump the Envoy version? Is it just because we want to publish updated example YAML?

**Comment by stevesloka** [comment]
> Is it just because we want to publish updated example YAML?

Yup basically. There have been times where we upgrade the go-control-plane protos and it changes how we pass Envoy config, so that meant a hard dep on a specific version of Envoy. However, these recently have not encountered that, so it's basically a no-op release for Contour.

**Comment by jpeach** [comment]
We discussed offline and agreed to not do a ` v1.6.2` release. Contour 1.7 will have the latest Envoy 1.15.0 version so there's nothing more to do here.
